### PR TITLE
simplify around code

### PR DESCRIPTION
**Pull request by fuentesloic**

### CODE REVIEW

File: static/components/PoiCard.js [hunk 1]
```diff
@@ -40,19 +40,19 @@ class PoiCard extends React.Component {
...
-        return `${distance} ${unit[0]}`
-      }
+    if(!distance){ 
+      return ''
+    } else if(distance > 1000) {
+      distance = Math.round(distance/1000)
+      return `${distance} ${unit[1]}`
+    } else {
+      return `${distance} ${unit[0]}`
     }
```
Comment (vinyll) [inline static/components/PoiCard.js]: What about:

```
renderDistance(distance, unit) {
  if(!distance) return ''
  distance = Math.round(distance/1000)
  return distance < 1000 ? `${distance} meters` : `${distance} km`
}
```

It seems more readable to me. What do you think?

Comment (fuentesloic) [reply static/components/PoiCard.js]: Oh I see ... so next level on writting javascript ^^
Sure ... that looks weird for now but I saw a lot of this "kind" of writting

Comment (vinyll) [reply static/components/PoiCard.js]: but the code is wrong.
Other suggestion:

```
renderDistance (distance) {
  if(!distance) return ''
  if(distance > 1000) {
    return `${Math.round(distance/1000)} km`
  }
  return `${distance} meters`
}
```

> I'm not using the `unit` array as I don't understand what it is needed for.

Comment (fuentesloic) [reply static/components/PoiCard.js]: I prefer this one !

**Review by vinyll — APPROVED** [review]

**Comment by vinyll** [comment]
Please squash ;)

**Pull request merged**